src/css/App.css

```diff
@@ -910,4 +910,8 @@ div.ReactTags__suggestions li mark {
 div.ReactTags__suggestions ul li.ReactTags__activeSuggestion {
     background: #b7cfe0;
     cursor: pointer;
-}
\ No newline at end of file
+}
+
+.form-card-body{
+  height: 600px;
```

Comment:
@BrianSerem, don't use heights in pixels, it's better to use percentages or checkout flexbox [here](https://www.youtube.com/watch?v=JJSoEo8JSnc).